tools/rofiTool/points.cpp

```diff
@@ -1,26 +1,39 @@
-#include "commands.hpp"
+#include <configuration/rofiworld.hpp>
+#include <dimcli/cli.h>
+
+#include "common.hpp"
 #include "rendering.hpp"
 
-#include <configuration/rofiworld.hpp>
 
-static auto command = Dim::Cli().command( "points" )
-    .desc( "Interactively preview a set of points defining the configuration" );
-static auto& inputFile = command.opt< std::string >( "<FILE>" )
-    .desc( "Specify source file" );
-static auto& showModules = command.opt< bool >( "modules" )
-    .desc( "Show with modules" );
+void points( Dim::Cli & cli );
 
-int points( Dim::Cli & /* cli */ ) {
-    auto configuration = parseConfiguration( *inputFile );
-    
-    if ( configuration.modules().size() == 0 )
-        throw std::runtime_error( "Configuration in '" + *inputFile + "' does not contain any modules to decompose into points" );
+static auto command = Dim::Cli()
+                              .command( "points" )
+                              .action( points )
+                              .desc( "Interactively preview a set of points "
+                                     "defining the rofi world" );
+static auto & inputFile = command.opt< std::string >( "<world_file>" )
+                                  .desc( "Specify rofi world source file" );
+static auto & showModules = command.opt< bool >( "modules" ).desc( "Show modules" );
 
-    affixConfiguration( configuration );
+void points( Dim::Cli & cli )
+{
+    auto world = parseRofiWorld( *inputFile );
+    if ( !world ) {
+        cli.fail( EXIT_FAILURE, "Error while parsing world", world.assume_error() );
+        return;
+    }
+    if ( world->modules().empty() ) {
+        cli.fail( EXIT_FAILURE, "Empty rofi world" );
+        return;
+    }
 
-    configuration.prepare().get_or_throw_as< std::runtime_error >();
+    affixRofiWorld( *world );
```

Comment:
Affixing a world in space (meaning adding a RigidJoint between some module and 0,0,0) should be done only if the world does not already contain a SpaceJoint ( if ( world->referencePoints().empty() ) ), otherwise .prepare() and/or .validate() should not need it.